packages/extensions/experience-platform-connector/src/handlers/addToCart.js

```diff
@@ -0,0 +1,60 @@
+const canHandle = event => event.type === 'CART_ADD_ITEM';
+
+const handle = (sdk, event) => {
+    const { payload } = event;
+
+    const {
+        cartId,
+        currencyCode,
+        priceTotal,
+        quantity,
+        name,
+        sku,
+        selectedOptions
+    } = payload;
+
+    const configurableOptions = selectedOptions
+        ? [
+              {
+                  optionLabel: selectedOptions.attribute,
+                  valueLabel: selectedOptions.value
+              }
+          ]
+        : null;
+
+    const cartItemContext = {
+        id: cartId,
+        prices: {
+            subtotalExcludingTax: {
+                value: priceTotal * quantity,
```

Comment:
Interesting that prices are calculated here. Is this value not available elsewhere?

Reply:
none that I could find